src/eureka/S4_generate_lightcurves/generate_LD.py

```diff
@@ -24,8 +28,31 @@ def exotic_ld(meta, spec):
     - July 2022, Eva-Maria Ahrer
         Initial version based on exotic_ld documentation.
     '''
+    
+    log.writelog("...using exotic-ld package...",
+                 mute=(not meta.verbose))
+    
+    # Check if exotic-ld directory includes csv files
+    exotic_ld_files = glob.glob(meta.exotic_ld_direc + '/**/*.flx',
+                                recursive=True)
+    if not exotic_ld_files:
+        raise AssertionError('Unable to find ancillary files.' +
+                             'Have you downloaded them? (see Zenodo link)')
+    
     # Set the observing mode
-    if meta.inst == 'miri':
+    custom_wavelengths = None
+    custom_throughput = None
+    
+    if meta.exotic_ld_file:
+        mode = 'custom'
+        log.writelog("Using custom throughput file " +
+                     meta.exotic_ld_file,
+                     mute=(not meta.verbose))
+        # load custom file
+        custom_data = pd.read_csv(meta.exotic_ld_file)
```

Comment:
Actually, it looks like at present you don't join `meta.exotic_ld_direc` to `meta.exotic_ld_file`. Make sure you pick one method of specifying `meta.exotic_ld_file` and stick with it everywhere

Reply:
Yes, I did that on purpose because I didn't want the user to have to put the custom file into the exotic_ld_direc e.g. I like my customary files to not be in the ancillary data folder.